test/Benchmarks/src/Startup/Startup.enso

```diff
@@ -8,16 +8,26 @@ polyglot java import java.io.File as Java_File
 type Data
     Value ~enso_bin:File ~empty_world:File ~hello_world:File ~import_world:File
 
-    bench_empty self =
-        self.startup [ "--run", self.empty_world.to_text ]
-
-    bench_hello self =
-        self.startup [ "--run", self.hello_world.to_text ]
-
-    bench_import self =
-        self.startup [ "--run", self.import_world.to_text ]
+    bench_empty self in_jvm:Boolean =
+        cmd = ["--run", self.empty_world.to_text]
+        case in_jvm of
+            True -> self.startup (["--jvm"] + cmd)
+            False -> self.startup cmd
+
+    bench_hello self in_jvm:Boolean =
+        cmd = ["--run", self.hello_world.to_text]
+        case in_jvm of
+            True -> self.startup (["--jvm"] + cmd)
+            False -> self.startup cmd
+
+    bench_import self in_jvm:Boolean =
+        cmd = ["--run", self.import_world.to_text]
+        case in_jvm of
+            True -> self.startup (["--jvm"] + cmd)
```

Comment:
- adding `in_jvm` argument to `self.startup` might remove some duplicated code

Reply:
Done in 4ae5572be2b368be986c9f8ffbbdf551edd7c8e1